PatientRegistrationApp/src/components/HouseholdContactInfo.vue

```diff
@@ -1,133 +1,146 @@
-                    label="Phone Type"
+					v-model="secondaryPhoneNumberType"
+					:items="phonetype"
+					label="Phone Type"
 				></v-select>
-        </v-col>
-</v-row>
-
-<v-row>
-
-		<v-col
-			cols="6"
-			sm="6"
-			md="3"
-		>
-		
-		<v-checkbox
-			v-model="checkbox"
-			label="I have no phone number"
-		></v-checkbox>
-		</v-col>
-
-</v-row>
-
-<v-row>
-		<v-col
-        cols="12"
-        sm="12"
-        md="6">
-			<v-text-field
-			:rules="['Required']"
-            label="Primary Email Address"
-			placeholder="johnsmith@email.com"
-			></v-text-field>
-        </v-col>
-
-		
-		<v-col
-        cols="12"
-        sm="12"
-        md="6">
-			<v-text-field
-            label="Secondary Email Address"
-			placeholder="johnsmith@email.com"
-			></v-text-field>
-        </v-col>
-
-</v-row>
-		<v-col
-			cols="6"
-			sm="6"
-			md="3"
-		>
-		
-		<v-checkbox
-			v-model="checkbox"
-			label="I have no email address"
-		></v-checkbox>
-		</v-col>
-
-<v-row>
-	<v-radio-group
-        v-model="test"
-        label="May we contact you regarding follow up vaccination information?"
-      >
-        <v-col align="right" cols="3" sm="3" md="3">
-          <v-radio label="Yes" value="yes"></v-radio>
-          <v-radio label="No" value="no"></v-radio>
-        </v-col>
-      </v-radio-group>
-	
-</v-row>
-
-</v-container>
+			</v-col>
+		</v-row>
+		<v-row>
+			<v-col cols="6" sm="6" md="3">
+				<v-checkbox
+					v-model="checkbox"
+					label="I have no phone number"
+				></v-checkbox>
+			</v-col>
+		</v-row>
+		<v-row>
+			<v-col cols="12" sm="12" md="6">
+				<v-text-field
+					required
+					:rules="emailRules"
+					label="Primary E-mail Address"
+					v-model="primaryEmail"
+				></v-text-field>
+			</v-col>
+			<v-col cols="12" sm="12" md="6">
+				<v-text-field
+					:rules="emailRules"
+					label="Secondary E-mail Address"
+					v-model="secondaryEmail"
+				></v-text-field>
+			</v-col>
+		</v-row>
+			<v-col cols="6" sm="6" md="3">
+				<v-checkbox
+					v-model="checkbox"
+					label="I have no email address"
+				></v-checkbox>
+			</v-col>
+		<v-row>
+			<v-radio-group
+				required
+				:rules="[v => !!v || 'This field is required']"
+				v-model="approval"
+				label="May we contact you regarding follow up vaccination information?"
+			>
+				<v-col align="right" cols="3" sm="3" md="3">
+					<v-radio label="Yes" value="yes"></v-radio>
+					<v-radio label="No" value="no"></v-radio>
+				</v-col>
+			</v-radio-group>
+		</v-row>
+	</v-container>
 </template>
 
 <script>
+import EventBus from '../eventBus'
 
   export default {
     name: 'SinglePatientContactInfo',
 	data () {
 		return { 
 			phonetype:['Cell', 'Landline', 'Other'],
-			radios: 'May we contact you for a follow up vaccination?'
+			radios: 'May we contact you for a follow up vaccination?',
+			emailRules: [
+				(v) =>
+				/^[\s]*$|.+@.+\..+/.test(v) ||
+				"E-mail must be in the format example@example.com",
+			],
+			primaryPhoneNumber: '',
```

Comment:
Do we have phone-number rules?

Reply:
MVS-257 and MVS-259 are for phone number validation (to do)

Reply:
Awesome thanks